optimum/onnxruntime/trainer.py

```diff
@@ -33,13 +33,28 @@
     is_fairscale_available,
 )
 
+from transformers.utils import is_accelerate_available
+from packaging import version
+
+if is_accelerate_available():
```

Comment:
From what I remember, this won't help solving the doc build dependency issue.

Reply:
@JingyaHuang I edited the dockerfile for docbuild accordingly.

Reply:
Why not, let's just keep it.